docs/wagtail-pages.md

```diff
@@ -0,0 +1,211 @@
+  Appears on the "Configuration" tab when editing a page. 
+
+Most fields will simply require a `FieldPanel` to be added to one of the sets of panels above. 
+StreamFields will require a `StreamFieldPanel`. 
+See the [Wagtail documentation for additional, more complex panel options](https://docs.wagtail.io/en/v1.13.4/topics/pages.html#editor-panels).
+
+For example, in our `BrowsePage` (used in the [database fields example above](#fields)),
+the `secondary_nav_exclude_sibling_pages` `BooleanField` 
+[is added to the `sidefoot_panels` as a `FieldPanel`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/browse_page.py#L63):
+
+```python
+from django.db import models
+from wagtail.wagtailadmin.edit_handlers import FieldPanel
+
+from v1.models.base import CFGOVPage
+
+
+class BrowsePage(CFGOVPage):
+    secondary_nav_exclude_sibling_pages = models.BooleanField(default=False)
+
+    # …
+
+    sidefoot_panels = CFGOVPage.sidefoot_panels + [
+        FieldPanel('secondary_nav_exclude_sibling_pages'),
+    ]
+```
+
+Because `secondary_nav_exclude_sibling_page` is a boolean field, 
+this creates a checkbox on the "Sidebar" tab when editing a page.
+Checking or unchecking that checkbox will set the value of `secondary_nav_exclude_sibling_pages` when the page is saved.
+
+In our `LandingPage` (used in the [StreamFields example above](#streamfields), 
+the `header` StreamField 
+[is added to the `content_panels` as a `StreamFieldPanel`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/landing_page.py#L31):
+
+```python
+from wagtail.wagtailadmin.edit_handlers import StreamFieldPanel
+from wagtail.wagtailcore.fields import StreamField
+
+from v1.atomic_elements import molecules
+from v1.models import CFGOVPage
+
+
+class LandingPage(CFGOVPage):
+    header = StreamField([
+        ('hero', molecules.Hero()),
+        ('text_introduction', molecules.TextIntroduction()),
+    ], blank=True)
+
+    # …
+
+    content_panels = CFGOVPage.content_panels + [
+        StreamFieldPanel('header'),
+        # …
+    ]
+```
+
+## Parent / child page relationships
+
+Wagtail provides two attributes to page models that enable 
+[restricting the three types of subpages or parent pages](https://docs.wagtail.io/en/v1.13.4/topics/pages.html#parent-page-subpage-type-rules) 
+a particular page model can have. 
+
+For example, in our [interactive regulations page models](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/regulations3k/models/pages.py#L138) 
+we have a `RegulationLandingPage` that can be created anywhere in the page tree. 
+`RegulationLandingPage`, however, can only have two types of pages created within it: 
+`RegulationPage` and `RegulationSearchPage`. 
+This parent/child relationship is expressed by setting `subpage_types` on `RegulationLandingPage`
+and `parent_page_types` on `RegulationPage` and `RegulationSearchPage` 
+to either the model classes themselves or to a model name in the form `app_label.ModelName`:
```

Comment:
This took me a minute to understand. Can we pick a preferred style and just say that?

Reply:
What do you mean? setting both `subpage_types` and `parent_page_types` is required.

Reply:
I'll add the two bullets from the Wagtail docs.

Reply:
I'm referring to the specific line I commented on: "to either the model classes themselves or to a model name in the form `app_label.ModelName`:". Can we pick one to prefer?

Reply:
Silly Django. Is there a reason that both methods need to exist, yet neither be preferred?

And would it be good to include the single quotes around `'app_label.ModelName'` for clarity?

Reply:
Yes, sometimes you can't import the module class directly, or sometimes you need to reference it before it's been defined. That's what the string-based lookup is for.

We could just recommend the string. I don't have strong feelings one way or the other, but both are possible and valid in the Django universe.

Reply:
Done in 15edfd8.